src/www/services_dnsmasq.php

```diff
@@ -371,6 +398,33 @@
                     <?= gettext('Log the results of DNS queries') ?>
                   </td>
                 </tr>
+                <tr>
+                  <td><a id="help_for_dns_forward_max" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Max number DNS queries");?></td>
+                  <td>
+                    <input name="dns_forward_max" type="text" id="dns_forward_max" size="6" placeholder="5000" <?=!empty($pconfig['dns_forward_max']) ? "value=\"{$pconfig['dns_forward_max']}\"" : "";?> />
+                    <div class="hidden" data-for="help_for_dns_forward_max">
+                      <?=gettext("The maximum number of forwarded DNS queries (default 5000)");?>
```

Comment:
I prefer the placeholder over embedding numbers in the help text

```suggestion
                      <?=gettext("The maximum number of forwarded DNS queries.");?>
```

Reply:
changed